header/js/typeahead.js

```diff
@@ -99,4 +107,23 @@ function trackSearchEvent (context) {
 	document.body.dispatchEvent(tracking);
 }
 
+// This is purposefully quite rubbish to be small, fast and target only the most-problematic chars.
+function latinize (text) {
+	const replacements = {
+		'a': ['à', 'á', 'â', 'ä', 'ã', 'å'],
```

Comment:
Can we somehow write an ES query to get a more complete list for our taxonomies e.s. I notice you don't have `ß`, which is very common in german